caffe2/observers/CMakeLists.txt

```diff
@@ -1,13 +1,10 @@
 if(USE_OBSERVERS)
   message(STATUS "Include Observer library")
-  set(GLOB profile_observer_files profile_observer_*.cc)
   set(Caffe2_CONTRIB_OBSERVERS_CPU_SRC
+    "${CMAKE_CURRENT_SOURCE_DIR}/profile_observer.cc"
     "${CMAKE_CURRENT_SOURCE_DIR}/time_observer.cc"
     "${CMAKE_CURRENT_SOURCE_DIR}/runcnt_observer.cc"
   )
-  set(Caffe2_CONTRIB_OBSERVERS_GPU_SRC
-    "${CMAKE_CURRENT_SOURCE_DIR}/profile_observer_gpu.cc"
-  )
```

Comment:
You can delete https://github.com/pytorch/pytorch/blob/master/caffe2/observers/CMakeLists.txt#L15-L16 as well